lib/rucio/tests/test_rse_protocol_gfal2_new.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
```

Comment:
looks good, but maybe use a more descriptive name than `..._new` ? this suggests that the old file can go away?